zeppelin-web/src/app/notebook/notebook.css

```diff
@@ -84,12 +84,10 @@
   color: #555;
   background: #fff;
   border: 1px solid #ccc;
-  border-radius: 4px;
+  border-radius: 0;
   -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
   box-shadow: inset 0 1px 1px rgba(0, 0, 0, .075);
-  -webkit-transition: border-color ease-in-out .15s, -webkit-box-shadow ease-in-out .15s;
```

Comment:
@corneadoug @prabhjyotsingh @Leemoonsoo Removed transition for both note and paragraph title, was causing the flicker while updating the title and the animation wasn't visible (Ref: Before screenshot)